dfffrontend/invoicemanualentry.js

```diff
@@ -12,301 +12,350 @@ const pSignature5 = document.getElementById("potpis_5");
 const result = document.getElementById("result");
 let hasDups = false;
 
+// Geting api key to local storage and seting for input value
+const apiKeyInput = document.getElementById("api_key");
```

Comment:
No need for this and the line below as evertyhing will be handled in the fetchApiKeyAndPrepareHeaders() method once the above comment is implemented.